client/src/actions/modelCommunication.js

```diff
@@ -0,0 +1,16 @@
+import * as api from "../api/index.js";
+import { PROCESS_AUDIO } from "./constants";
+
+export const processCandidateAnswer = (audioBase64) => async (dispatch) => {
```

Comment:
It will not work as expected. 
some references - https://www.npmjs.com/package/react-media-recorder